logger.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import logging.handlers
+
+class LevelFilter(logging.Filter):
+    def __init__(self, level):
+        self.level = level
+
+    def filter(self, record):
+        return record.levelno < self.level
+
+class Logger:
+    @staticmethod
+    def create_logger(log_filename, logger_name, level_name):
+        logger = logging.getLogger(logger_name)
+        logging_levels = {'CRITICAL': 50, 'ERROR': 40, 'WARNING': 30, 'INFO': 20, 'DEBUG': 10, 'NOTSET': 0}
+        logger.setLevel(logging_levels[level_name])
+        logger.propagate = False
```

Comment:
can you explain why you set this to false?

Reply:
I think setting it to true will attach the handler with logger and it will log same thing twice